README.md

```diff
@@ -3,24 +3,62 @@
 
 A CKeditor plugin that makes adding binder enabled pre tags easy.
 
-### Installation
+## Installation
 
 ```
 yarn install
 ```
 
-### Start Dev Server
+## Start Dev Server
 
 ```
 yarn start
 ```
 
-### Build Prod Version
+## Build Prod Version
 
 ```
 yarn build
 ```
 
-### Linter
+## Linter
 
 This repo follows Airbnb Javascript guide, see [here](https://github.com/airbnb/javascript) for more information.
+
+
+## Usage with LibreTexts
+
+CKEditor is a rich text editor which enables the user to write content directly inside web pages or online applications. Through the Binder Plugin, authors of textbooks have the option to include code within in their textbook pages to create a more interactive experience for their students.
+
+
+
+
+Currently the plugin supports Python 3, Julia, R, Octave, and SageMath. Additionally, all languages with the exception of SageMath support syntax highlighting for an easier scripting experience. 
+
+
+
+### Step-by-Step Process to use CKEditor Binder Plugin
+
+
+#### Creating a new page
+Authors have the option to add code to a new page.
+
+![](tutorialVisuals/newPage.gif)
+
+#### Editing a exisiting page
+Additionally, authors can also add code to pages that they have already created. To do so, simply click on the "Edit" button.
+![](tutorialVisuals/editOld.gif)
+
+#### Locating the Plugin and Adding Code
+The Binnder Logo is the last button on the navigation bar of the text editor.  
+
+![](tutorialVisuals/ckePlugin.png)  
+
+#### Add Code
```

Comment:
```suggestion
#### Add code
```
Capitalization consistency, since the first two level-4 headers were in sentence case.